### DISCUSSION TITLE
orgCd 추출 불가

### DISCUSSION THREAD

**Discussion by julian1z** [changdoc/naver-vaccine-macro#147] [General]
네이버측에서 org코드를 숨긴 것 같네요
기존에 추출해놓은 url은 작동합니다

**Comment by changdoc** [comment]
야간이라 그런지 graphql response에서 org 코드를 포함하는 상위 객체 VaccineQuantity가 null인 상태네요.
![image](https://user-images.githubusercontent.com/1879494/127529873-ca68a4f5-3452-4c65-9ded-c0e728ae5e9f.png)

내일 오전에도 null인거면 매크로 대응하기 위해 네이버에서 조치를 취한 것으로 보입니다.

**Reply by changdoc** [reply]
현재 시각 7월30일 오전 9시14분
정상이네요
<img width="475" alt="스크린샷 2021-07-30 오전 9 13 11" src="https://user-images.githubusercontent.com/1879494/127581098-7d5fc8b0-c544-41d8-a1ea-d041bb8a5112.png">
